tests/Message/PurchaseRequestTest.php

```diff
@@ -75,4 +78,3877 @@ public function testSetConsumerLanguage()
         $this->request->setConsumerLanguage('en');
         $this->assertSame('002', $this->request->getConsumerLanguage());
     }
+
+    // cardholder
+    public function testCardholderInitializer()
+    {
+        $this->request->initialize([
+            'merchantId' => '999008881',
+            'terminalId' => '871',
+            'amount'     => '1.45',
+            'currency'   => 'EUR',
+            'cardholder' => "Mr Test"
```

Comment:
Worth pulling out the common parameters used by `setUp()` into a class property?

Reply:
Done.